controllers/etcd_controller.go

```diff
@@ -883,6 +889,27 @@ func (r *EtcdReconciler) removeFinalizersToDependantSecrets(etcd *druidv1alpha1.
 				}
 			}
 		}
+
```

Comment:
Why copy-paste like this? Why not run this in a loop for all the secrets what need to be updated?